projects/elfutils/fuzz-dwfl-core.c

```diff
@@ -38,7 +38,7 @@ int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
 	assert(fd >= 0);
 
 	n = write(fd, data, size);
-	assert(n == size);
+	assert(n == (ssize_t) size);
 
 	offset = lseek(fd, 0, SEEK_SET);
 	assert(offset == 0);
```

Comment:
I don't think it's as important to remove existing zips, the space is already taken up in the git history (I guess removing zips makes --depth 1 faster). But not adding new ones is good policy